app/views/users/profile.html.erb

```diff
@@ -46,6 +45,15 @@
 			<!--<%= f.input :shirt_size, :as => :select, :collection => %w[XS S M L XL XXL 3XL] %>-->
 		<% end -%>
 	<% end -%>
+	<%- if @person.is_a?(Student) || @person.is_a?(Participant) -%>
```

Comment:
For some reason, this block doesn't seem to be loading when I log in as a student.